flybywiresim/a32nx
# Enable auto pitot heating while engines are running

Fixes #245, #969 and partially #728 and few others that were closed.

**Summary of Changes**

Pitot heating should now be automatically turned on when at least one engine is running and the switch is in auto mode, be it on the ground or in the air.

**Screenshots (if necessary)**


N/A

**References**

[http://www.a320dp.com/A320_DP/ice-rain/sys-11.1.2.html](http://www.a320dp.com/A320_DP/ice-rain/sys-11.1.2.html)
[https://www.youtube.com/watch?v=O170fbsUTgU](https://www.youtube.com/watch?v=O170fbsUTgU)




**Additional context**

A bit tricky to test for non tech-savvy, requires reading the `Simplane.GetSimVarValue("PITOT HEAT", Bool);`, so you don't have to wait for the plane to freeze somewhere. Without the fix that variable will be 0 if the PROBE HEAT button is not pressed.

## How to download this PR for QA
Every new commit to this PR will cause a new A32NX artifact to be created and uploaded. The build.py will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the Checks tab on the PR
1. On the right side, slick on the Artifacts drop down and click the A32NX link
